cypress/integration/instance/InstanceEdit.ts

```diff
@@ -50,5 +50,65 @@ onlyOn(isEnvironmentType(EnvType.Mocked), () => {
             });
         });
     });
+
+    it("Switch from 'Webhook' to 'Ignore' Error handler", () => {
+      //Open 'Instance one' page
+      cy.ouiaId("Instances list table", "PF4/Table")
+        .ouiaId("Instance one", "PF4/TableRow")
+        .within(() => {
+          cy.get(
+            "td:first a[data-testid='tableInstances-linkInstance']"
+          ).click();
+        });
+
+      // Switch to error handling tab
+      cy.ouiaId("error-handling", "PF4/TabButton").click();
+
+      // Wait tab is loaded
+      cy.get(".pf-c-description-list__term").should("have.length", 5);
+      // Once tab is loaded start edit mode
+      cy.ouiaId("error-handling", "PF4/TabContent").within(() => {
+        cy.ouiaId("edit", "PF4/Button")
+          .should("have.attr", "aria-disabled")
+          .and("equal", "false");
+        cy.ouiaId("edit", "PF4/Button").click();
+      });
+
+      // Change error handling strategy to 'Ignore'
+      cy.ouiaId("error-handling-method-selector", "PF4/Select").within(() => {
+        cy.get("button").should("not.have.attr", "disabled");
+      });
```

Comment:
Similar question. Is there any reason for using within? This construction should work:
```
cy.ouiaId("error-handling-method-selector", "PF4/Select").find("button").should("not.have.attr", "disabled");
```

Reply:
well, `within` is my personal preference, is there any cypress recommendation/good practice to use `find` instead of `within`?

Reply:
I do not think so. It surprised me because I usually use `find` when I search 1 element and `within` when I need to assert more than 1 element.